content/zh-cn/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -183,7 +206,10 @@ that are most appropriate for your configuration, do the following:
       kubectl label --dry-run=server --overwrite ns --all \
       pod-security.kubernetes.io/enforce=restricted
       ```
-      <!-- The output is similar to this: -->
+
+      <!--
+      The output is similar to:
+      -->
       输出类似于：
```

Comment:
```suggestion
      <!--
      The output is similar to:
      -->
      
      输出类似于：
      
```
This line has problem:

<img width="684" alt="image" src="https://user-images.githubusercontent.com/79828097/224879851-75da1a96-ef68-4ad9-b22d-bc5d33500072.png">